electrical/breakbeam.py

```diff
@@ -0,0 +1,47 @@
+import RPi.GPIO as GPIO
+import time
+
+class Breakbeam:
+    ACCEPTABLE_TIME = 50 # in seconds
```

Comment:
Why are we choosing 50 seconds to be acceptable - Seems a bit arbitrary? May not necessarily be a problem, however, until we have a better reference, lets keep any eye out.

Reply:
^^ if this is a temp/experimental value maybe we can make a note in comments

Reply:
It is a bit arbitrary rn, couldn't think of a better reference yeah. I'll add a note. Should I make a new commit with the changes?